pkg/ec2helper/types.go

```diff
@@ -39,6 +39,7 @@ type EC2Svc interface {
 	DeleteSecurityGroup(input *ec2.DeleteSecurityGroupInput) (*ec2.DeleteSecurityGroupOutput, error)
 	CreateLaunchTemplate(input *ec2.CreateLaunchTemplateInput) (*ec2.CreateLaunchTemplateOutput, error)
 	DeleteLaunchTemplate(input *ec2.DeleteLaunchTemplateInput) (*ec2.DeleteLaunchTemplateOutput, error)
+	DescribeLaunchTemplates(input *ec2.DescribeLaunchTemplatesInput) (*ec2.DescribeLaunchTemplatesOutput, error)
```

Comment:
We already include the paginated version of this API, `DescribeLaunchTemplatesPages` (line 26), as well as the corresponding [`ec2helper_mock` implementation](https://github.com/GavinBurris42/aws-simple-ec2-cli/blob/4f9fd861b22e8f814a059799fa0b371848df81fb/test/testhelper/ec2helper_mock.go#L99-L124). It's slightly more complicated to use because it has a callback function, but I'd prefer to re-use it rather than adding a new API, for simplicity's sake. On the other hand, if there's a good reason to add it, I can be persuaded. What do you think?

Reply:
The Describe Launch Template Pages only returns an error whereas the Describe Launch Templates returns launch templates which I need to be able to test the proper functionality of the methods.